roles/satellite-clone/files/disassociate_capsules.rb

```diff
@@ -11,42 +12,39 @@ def prepare_hammer_cmd(command)
 end
 
 def run_hammer_cmd(command)
-  command = prepare_hammer_cmd(command)
-  `#{command}`
+  `#{prepare_hammer_cmd(command)}`
 end
 
-def get_info_from_hammer(command, column=1)
-  bash_parse = " | grep -v \"Warning:\" | tail -n+2 | awk -F, {'print $#{column}'}"
-  run_hammer_cmd(command + bash_parse)
+def get_ids_from_hammer(command, search = nil)
+  JSON.parse(run_hammer_cmd("--output=json " + command + (search ? " --search '#{search}'" : "")).map do |obj|
+    obj["Id"]
```

Comment:
this is funny (well, actually not) but I recall this sometimes being `Id` and sometimes `ID`, depending on the hammer version…

Reply:
https://github.com/Katello/hammer-cli-katello/commit/416e7f780acc3a958e8c9e0ea0d4e565766002f3